core/lib/zksync_core/src/l1_gas_price/mod.rs

```diff
@@ -10,24 +10,11 @@ mod gas_adjuster;
 mod main_node_fetcher;
 pub mod singleton;
 
-/// Abstraction that provides information about the L1 gas price currently
-/// observed by the application.
-pub trait L1GasPriceProvider: fmt::Debug + 'static + Send + Sync {
-    /// Returns a best guess of a realistic value for the L1 gas price.
-    /// Return value is in wei.
-    fn estimate_effective_gas_price(&self) -> u64;
-
-    /// Returns a best guess of a realistic value for the L1 pubdata price.
```

Comment:
Was it used for previous fee model only?

Reply:
No, `estimate_effective_gas_price` seems to be invoked for both old and new model, and `estimate_effective_pubdata_price` seems to be invoked only for the new model.
They still exist, but now they're just methods on `GasAdjuster`, not a part of some trait.